### PR TITLE
Add contribution and reviewer checklist to issue

### PR DESCRIPTION
**Pull request by anthonyvetter**
In order to make the contribution process more visible, I have added abbreviated versions of what is available on the Wiki to the content contribution issue template.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/content-proposal.md [hunk 1]
```diff
@@ -17,8 +17,36 @@ A short summary of the audience. Include:
...
-- [ ] Content Series: To include multiple levels of content (Note: Series pieces should either be able to be consumed independently, or very explicit in the title, and with links in the body of the content that they are part of a series)
+Level of content as it pertains to the topic proposed. Delete those not applicable.
+- 101: "Awareness / Exploration"
+- 201: "Prototyping / Deploying"
+- 301: "Scaling
+- Content Series: To include multiple levels of content (Note: Series pieces should either be able to be consumed independently, or very explicit in the title, and with links in the body of the content that they are part of a series)
+
+## Contributor Checklist
+1. [x] I have added a card to the [GitHub Content Backlog](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7) and/or moved it to the proper column. See [Content Prewriting and Ideation](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Prewriting-Ideation-(GitHub-and-Google-Docs-Method))
+2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and made is visible with comment rights to everyone.
```
Comment (joemoore) [inline .github/ISSUE_TEMPLATE/content-proposal.md:28]:
```suggestion
2. [ ] **If contributing via Google Docs**: I have added a link to my document to this issue, and set the document's Share settings to "Anyone on the internet with this link can comment".
```

Comment (joemoore) [reply .github/ISSUE_TEMPLATE/content-proposal.md:28]: Screen_Shot_2021-08-04_at_3_19_22_PM

Consider referencing this or similar screenshot.

File: .github/ISSUE_TEMPLATE/content-proposal.md [hunk 2]
```diff
@@ -17,8 +17,36 @@ A short summary of the audience. Include:
...
+4. [ ] The front matter (metadata) is filled out completely (e.g. description, topics, tags, author, etc - as appropriate for the content type). See [Drafting Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Drafting-Content-(GitHub-Method))
+5. [ ] My content has been reviewed by my team for technical accuracy. See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))
+6. [ ] My content has beed reviewed and edited by a copy editor. See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))
+7. [ ] **If applicable**: My content has gone through a legal review (**VMware Employees Only**). See [Editing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Editing-Content-(GitHub-and-Google-Docs-Method))
+8. [ ] I have tested my content locally and everything passes. See [Testing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Testing-Content-(GitHub-Method))
```
Comment (joemoore) [inline .github/ISSUE_TEMPLATE/content-proposal.md:34]:
```suggestion
8. [ ] **If applicable**: I have tested my content locally and everything passes. See [Testing Content](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Testing-Content-(GitHub-Method))
```

Comment (joemoore) [reply .github/ISSUE_TEMPLATE/content-proposal.md:34]: As not all content is "testable".

File: .github/ISSUE_TEMPLATE/content-proposal.md [hunk 3]
```diff
@@ -17,8 +17,36 @@ A short summary of the audience. Include:
...
+9. [ ] **Optional**: I have filled out the [Social Media request form](https://docs.google.com/forms/d/e/1FAIpQLSfyWYEL6XvDRfMRW1GBfYKV4Aq3Li2EzBz0HY8fE37Y2tSoRw/viewform) (optional) (**VMware Employees Only**). See [Next Steps](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Next-Steps)
+10. [ ] **If applicable**: I have created an author page and opened a pull request to get it added. See [Next Steps](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Next-Steps)
+
+
+## Reviewer Checklist
+
+* [ ] Review if the content is a good fit for the Tanzu Developer Center.
```
Comment (joemoore) [inline .github/ISSUE_TEMPLATE/content-proposal.md:41]:
```suggestion
* [ ] Review if the content is a good fit for the Tanzu Developer Center. See [Content Prewriting and Ideation](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Prewriting-Ideation-(GitHub-and-Google-Docs-Method))
```

File: .github/ISSUE_TEMPLATE/content-proposal.md [hunk 4]
```diff
@@ -17,8 +17,36 @@ A short summary of the audience. Include:
...
+10. [ ] **If applicable**: I have created an author page and opened a pull request to get it added. See [Next Steps](https://github.com/vmware-tanzu/tanzu-dev-portal/wiki/Contribution,-Next-Steps)
+
+
+## Reviewer Checklist
+
+* [ ] Review if the content is a good fit for the Tanzu Developer Center. 
+* [ ] There this GitHub issue has been added to the [content backlog project](https://github.com/vmware-tanzu/tanzu-dev-portal/projects/7) 
+* [ ] Ensure with the author that the content has been reviewed for technical accuracy.
+* [ ] Ensure with the author that the content has been through copy editing.
+* [ ] I**If applicable**: Ensure with the author that the content has been through Legal review.
```
Comment (joemoore) [inline .github/ISSUE_TEMPLATE/content-proposal.md:45]:
```suggestion
* [ ] **If applicable**: Ensure with the author that the content has been through Legal review.
```

**Comment by djschny** [comment]
Thanks for updating this. Instead of updating the initial submission template with Google Docs information, I would suggest we make a separate template. So we would have:

* Tanzu Content Proposal
* Tanzu Content Proposal - Google Docs Method

The reason I say this is because a person who doesn't feel comfortable with GitHub will have the following characteristics:

* Most likely be sent a link directly to the file a new issue and not self-discover it
* Seeing all the markdown appear with the textual checkboxes, and formatting as opposed to rendered will be intimidating.

Therefore on the Google Docs method one it can be simple and in plain text ask for basic information and the Google Docs link, etc.

**Comment by anthonyvetter** [comment]
@djschny Ya I thought about that too. My thinking was that folks contributing via GDocs probably wouldn't open their own issue either. So we could make due with 1 template. The messaging thus far has been that GDocs contributors fill out our Google Form, and that's pretty much it. The reviewer that picks up the contribution would open the issue, etc. In the event the reviewer caught wind of a piece of content prior to it being submitted (which is not the common case) they would open the issue at that time.

**Comment by djschny** [comment]
>  My thinking was that folks contributing via GDocs probably wouldn't open their own issue either.

If that is the case then why even bother to put Google Docs info in the template?

**Pull request merged**